appengine/mailgun/README.md

```diff
@@ -0,0 +1,62 @@
+## Mailgun on Google App Engine
+
+> [Mailgun](https://www.mailgun.com/): The Email Service For Developers
+
+### Sign up for Mailgun
+
+1. Sign up for a [Mailgun account](https://mailgun.com/signup).
+1. Add a [new domain](https://mailgun.com/app/domains).
+1. Find your API key in your new domain's settings.
+
+### Create a new Node.js app
+
+This example is going to use [Express.js](http://expressjs.com).
+
+### Configure
+
+Create an `app.yaml` in the root of your application with the following
+contents:
+
+```yaml
+runtime: nodejs
+vm: true
+api_version: 1
+env_variables:
+  PORT: 8080
+```
+
+Create a `config.js` in the root of your application with the following
+contents:
+
+```yaml
+module.exports = {
+  mailgunApiKey: 'your-mailgun-api-key'
+};
+```
+
+### Start the app locally
+
+```
+$ npm start
+```
+
+Now visit http://localhost:8080 and try sending yourself an email.
+
+### Deploy
+
+For convenience, you can use an npm script to run the `gcloud` command. Add
+these lines to your `package.json` file:
+
+```json
+"scripts": {
+  "start": "node ./bin/www",
+  "deploy": "gcloud preview app deploy app.yaml --promote --project <your-project-id>"
```

Comment:
That's true, but this command can't fail (for lack of specifying a project). It doesn't have any dependency on them having set the gcloud project correctly, gives the user better insight into what's happening, and is consistent with the other `appengine/` samples.